python/lsst/ts/audiotrigger/read_serial_temp_scanner.py

```diff
@@ -130,19 +130,20 @@ def __init__(
         self.log = log
         self.simulation_mode = bool(simulation_mode)
         self.sensor_dict = {}
-        self.sample_wait_time = 5
+        self.sample_wait_time = 1
```

Comment:
In line 123, why is data a class variable? It can be an instance variable, right? If yes then please modify that.